.changes/unreleased/Under the Hood-20240112-230236.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Under the Hood
+body: Implement unit testing in CI
```

Comment:
Generally we don't have changelogs for CI changes.

Reply:
Fair, force of habit. We can clean that up afterwards.